### PR TITLE
19w12a mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/entity/ai/brain/Brain.mapping [hunk 1]
```diff
@@ -21,7 +21,8 @@ CLASS aiw net/minecraft/entity/ai/brain/Brain
 	METHOD b forget (Laod;)V
 	METHOD b setActivity (Lawe;)V
 	METHOD b stopAllTasks (Lvd;Laih;)V
+	METHOD c getMemory (Laod;)Ljava/util/Optional;
```
Comment (Gegy) [inline mappings/net/minecraft/entity/ai/brain/Brain.mapping:24]: What's the difference in implementation between `getMemory` and `getOptionalMemory`?

File: mappings/net/minecraft/client/audio/SoundSystem.mapping [hunk 2]
```diff
@@ -0,0 +1,41 @@
+CLASS dwm net/minecraft/client/audio/SoundSystem
+	FIELD a MARKER Lorg/apache/logging/log4j/Marker;
+	FIELD b LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD c unknownSounds Ljava/util/Set;
+	FIELD d loader Ldwp;
+	FIELD e settings Lcue;
+	FIELD f initialized Z
+	FIELD g soundEngine Lcrw;
```
Comment (Gegy) [inline mappings/net/minecraft/client/audio/SoundSystem.mapping:8]: The difference between `SoundSystem` and `SoundEngine` isn't too clear

File: mappings/net/minecraft/entity/ai/brain/Brain.mapping (hunk 1 above)
Comment (Runemoro) [reply mappings/net/minecraft/entity/ai/brain/Brain.mapping:24]: `getMemory` is the most used merhod and will throw an exception if there is no memory. `getOptionalMemory` has just two usages, and will return Optional.empty() if it's missing.

Comment (Runemoro) [reply mappings/net/minecraft/entity/ai/brain/Brain.mapping:24]: `getMemory` is the most used merhod and will throw an exception if there is no memory. `getOptionalMemory` has just two usages, and will return Optional.empty() if it's missing.

Comment (Gegy) [reply mappings/net/minecraft/entity/ai/brain/Brain.mapping:24]: `getMemory` also returns an `Optional`, though

**Comment by modmuss50** [comment]
Merging due to update, please open an issue or another PR if you have issues with this.

**Pull request merged**